src/category_theory/preadditive/additive_functor.lean

```diff
@@ -44,30 +43,28 @@ section
 variables {C D : Type*} [category C] [category D] [preadditive C]
   [preadditive D] (F : C ⥤ D) [functor.additive F]
 
-@[simp]
-lemma map_zero {X Y : C} : F.map (0 : X ⟶ Y) = 0 :=
-functor.additive.map_zero'
-
 @[simp]
 lemma map_add {X Y : C} {f g : X ⟶ Y} : F.map (f + g) = F.map f + F.map g :=
 functor.additive.map_add'
 
+/-- `F.map_add_hom` is an additive homomorphism whose underlying function is `F.map`. -/
+@[simps]
```

Comment:
A quick test suggests that `@[simp]` might make more sense here: with `@[simp]`, the simplifier turns the term `(F.map_add_hom : (X ⟶ Y) →+ _).to_fun` into `F.map`, whereas with `@[simps]` we only get `⇑(F.map_add_hom)`.